front/src/modules/timetableItem/components/ManageTimetableItem/helpers/buildPacedTrainException.ts

```diff
@@ -125,14 +109,13 @@ export function generatePacedTrainException(
   const pacedTrainStartTime = new Date(updatedOccurrence.start_time);
   pacedTrainStartTime.setMilliseconds(0);
 
-  if (!isEqual(originalStartTimeToTest, pacedTrainStartTime)) {
+  if (occurrenceIndex === null || !isEqual(originalStartTimeToTest, pacedTrainStartTime)) {
```

Comment:
Why add an `occurrenceIndex === null` check here?

An added exception (ie. one without an `occurrenceIndex`) is at least an exception for `start_time`. This change breaks this.